src/configure-tests/symbol-tests

```diff
@@ -3,3 +3,5 @@ sys_umount
 sys_oldumount
 sys_call_table
 printk
+blk_mq_submit_bio
+kfree
```

Comment:
printk became a macro, so it can be removed from this file asit always has addr zero, other have to be checked

Reply:
resolved